docs/Concepts/Node-Types.md

```diff
@@ -37,19 +37,28 @@ genesis block.
 
 !!! important
 
-    Fast synchronization is the default for named networks specified with
-    the [`--network` option](../Reference/CLI/CLI-Syntax.md#network), except for the `dev` development
-    network.
-
-    Fast synchronization is also the default if Ethereum mainnet is being connected to by not specifying
-    the [`--network`](../Reference/CLI/CLI-Syntax.md#network) and [`--genesis-file`](../Reference/CLI/CLI-Syntax.md#genesis-file)
+    Fast synchronization is the default for named networks specified using the
+    [`--network`](../Reference/CLI/CLI-Syntax.md#network) option, except for the `dev` development network.
+    It's also the default if connecting to Ethereum MainNet by not specifying the
+    [`--network`](../Reference/CLI/CLI-Syntax.md#network) and [`--genesis-file`](../Reference/CLI/CLI-Syntax.md#genesis-file)
     options.
 
 !!! note
 
-    To improve performance, or if the fast sync is taking a long time to complete, use
+    When fast synchronizing, block numbers increase until close to the head block, then the synchronization stalls for
+    some time depending on world state size.
+    For example, MainNet may take several days or more to fast synchronize.
+    Fast synchronization time may increase because Besu picks new pivot blocks, or because peers prune the world state
+    before it completes downloading.
+
+    To improve performance, or if the fast synchronization is taking a long time to complete, use
     [`--fast-sync-min-peers`](../Reference/CLI/CLI-Syntax.md#fast-sync-min-peers) to increase the number of peers to at least 8.
```

Comment:
I'm not sure `--fast-sync-min-peers` will help improve performance here. My understanding is that it causes the sync to pause if there aren't at least 8 peers which could make it take longer, but I could have misunderstood the option.

I'd have expected that increasing the total number of peers with `--max-peers` would have helped by making more peers available to request world state from and make it more likely one of them still hasn't pruned world state. Would need confirmation from someone more up to date on this than me though.